geode-cq/src/distributedTest/java/org/apache/geode/internal/cache/tier/sockets/DurableClientTestBase.java

```diff
@@ -216,33 +208,35 @@ void verifyDurableClientPresence(int durableClientTimeout, String durableClientI
     });
   }
 
+  void waitUntilHARegionQueueSizeIsZero(VM serverVM) {
+    serverVM.invoke(() -> await().atMost(60, SECONDS)
+        .until(() -> getClientProxy().getHARegionQueue().size() == 0));
+  }
+
   public void closeDurableClient() {
     durableClientVM.invoke((SerializableRunnableIF) CacheServerTestUtil::closeCache);
   }
 
   public void disconnectDurableClient(boolean keepAlive) {
-    printClientProxyState("Before");
+    // printClientProxyState("Before");
     durableClientVM.invoke("close durable client cache",
         () -> CacheServerTestUtil.closeCache(keepAlive));
-    await()
-        .until(CacheServerTestUtil::getCache, nullValue());
-    printClientProxyState("after");
+    await().until(CacheServerTestUtil::getCache, nullValue());
+    // printClientProxyState("after");
```

Comment:
commented out code here and above.

Reply:
I commented out this code to reduce logging. I want to leave it for whomever wanted it there in the first place because I am assuming I am stepping on someone's toes.

Reply:
Step on them. Tests should not log.

Reply:
Will do.